build.gradle

```diff
@@ -40,6 +40,7 @@ dependencies {
     implementation "io.opentelemetry:opentelemetry-exporter-otlp-logs:$otelVersion-alpha"
     implementation "io.opentelemetry:opentelemetry-exporter-logging" // only for debug
     runtimeOnly("io.opentelemetry.instrumentation:opentelemetry-logback-appender-1.0:$otelVersion-alpha")
+    runtimeOnly "io.opentelemetry.instrumentation:opentelemetry-resources:$otelVersion-alpha"
```

Comment:
The service name is also detected [here](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/resources/library/src/main/java/io/opentelemetry/instrumentation/resources/JarServiceNameDetector.java):

I think the [manifest](https://github.com/grafana/grafana-opentelemetry-starter/blob/4e44e25901085d5014be2a99944ef84cb90dd150/src/main/java/com/grafana/opentelemetry/GrafanaProperties.java#L33) should have priority over the jar name - can you check if that's the case?

Reply:
might also make sense to describe how the service.name is created from both the manifest and the jar name there.

Reply:
Took a closer look.  Below is the order of precedence for service name

1.  OTEL_SERVICE_NAME 
2.  OTEL_RESOURCE_ATTRIBUTES
     a. `service_name` key:value
     b. spring application name
     c. manifest 

If none of the above are set the service_name shows as `unknown_service:java`